packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -189,10 +189,10 @@ function persistedProfileMigrationToV11(
     if (!existingProfile?.network) {
         if (existingProfile?.networkType) {
             const networkId = getNetworkIdFromOldNetworkType(existingProfile?.networkType)
-            const network = NETWORK?.[networkId]
+            const network = NETWORK_METADATA_MAP[networkId] as IPersistedNetwork
```

Comment:
Wouldn't need to cast if NETWORK_METADATA_MAP was a map of default peristed networks